teamcity: failed tests on master: test/TestShowSessions

The following tests appear to have failed:

[#839099](https://teamcity.cockroachdb.com/viewLog.html?buildId=839099):


```
--- FAIL: test/TestShowSessions (0.500s)
show_test.go:684: unexpected number of running sessions: 2, expected 1.
	Active sessions (results might have changed since the test checked):
	app: "", query: "SELECT active_queries, last_active_query, application_name FROM [SHOW CLUSTER SESSIONS]", last query: SELECT node_id, (now() - session_start)::FLOAT FROM [SHOW CLUSTER SESSIONS] WHERE application_name NOT LIKE 'internal-%'
------- Stdout: -------
W180815 17:57:07.840555 154357 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 17:57:07.854188 154357 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 17:57:07.854325 154357 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:07.854339 154357 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:07.856789 154357 server/config.go:496  [n?] 1 storage engine initialized
I180815 17:57:07.856823 154357 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 17:57:07.856833 154357 server/config.go:499  [n?] store 0: in-memory, size 0 B
I180815 17:57:07.869104 154357 server/node.go:373  [n?] **** cluster 2b1f25af-ff88-4657-b6e8-c5105666374d has been created
I180815 17:57:07.869185 154357 server/server.go:1398  [n?] **** add additional nodes by specifying --join=127.0.0.1:40607
I180815 17:57:07.869412 154357 gossip/gossip.go:383  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:40607" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1600-gaa1d0d2" started_at:1534355827869343827 
I180815 17:57:07.870450 154357 storage/store.go:1506  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180815 17:57:07.872955 154357 server/node.go:476  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=1, queries=0.00, writes=0.00, bytesPerReplica={p10=7103.00 p25=7103.00 p50=7103.00 p75=7103.00 p90=7103.00 pMax=7103.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180815 17:57:07.873118 154357 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180815 17:57:07.873222 154357 server/node.go:697  [n1] connecting to gossip network to verify cluster ID...
I180815 17:57:07.873258 154357 server/node.go:722  [n1] node connected via gossip and verified as part of cluster "2b1f25af-ff88-4657-b6e8-c5105666374d"
I180815 17:57:07.873293 154357 server/node.go:546  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180815 17:57:07.873769 154357 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:07.873829 154357 server/server.go:1833  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 17:57:07.878003 154561 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:07.890063 154357 server/server.go:1535  [n1] starting https server at 127.0.0.1:33349 (use: 127.0.0.1:33349)
I180815 17:57:07.890233 154357 server/server.go:1537  [n1] starting grpc/postgres server at 127.0.0.1:40607
I180815 17:57:07.890267 154357 server/server.go:1538  [n1] advertising CockroachDB node at 127.0.0.1:40607
I180815 17:57:07.918927 154596 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180815 17:57:07.925720 154569 storage/replica_command.go:280  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180815 17:57:07.934915 154472 storage/replica_command.go:280  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180815 17:57:07.945613 154612 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180815 17:57:07.952062 154247 storage/replica_command.go:280  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180815 17:57:07.960984 154627 storage/replica_command.go:280  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180815 17:57:07.977602 154636 storage/replica_command.go:280  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180815 17:57:07.978699 154476 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
W180815 17:57:07.984580 154624 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "split" id=ca8ea57d key=/Local/Range/System/tsd/RangeDescriptor rw=true pri=0.04340252 iso=SERIALIZABLE stat=PENDING epo=0 ts=1534355827.977704188,0 orig=1534355827.977704188,0 max=1534355827.977704188,0 wto=false rop=false seq=1
I180815 17:57:07.988519 154659 storage/replica_command.go:280  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180815 17:57:08.003157 154604 storage/replica_command.go:280  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180815 17:57:08.009692 154252 storage/replica_command.go:280  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180815 17:57:08.016497 154649 storage/replica_command.go:280  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180815 17:57:08.042777 154256 storage/replica_command.go:280  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180815 17:57:08.046632 154687 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
I180815 17:57:08.071168 154739 storage/replica_command.go:280  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180815 17:57:08.073688 154656 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180815 17:57:08.078018 154746 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180815 17:57:08.078638 154753 storage/replica_command.go:280  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180815 17:57:08.079904 154357 server/server.go:1620  [n1] done ensuring all necessary migrations have run
I180815 17:57:08.079943 154357 server/server.go:1623  [n1] serving sql connections
I180815 17:57:08.086056 154700 storage/replica_command.go:280  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180815 17:57:08.086571 154763 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180815 17:57:08.092285 154835 storage/replica_command.go:280  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180815 17:57:08.098779 154765 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:40607} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1600-gaa1d0d2 StartedAt:1534355827869343827 LocalityAddress:[]} ClusterID:2b1f25af-ff88-4657-b6e8-c5105666374d StartedAt:1534355827869343827 LastUp:1534355827869343827}
I180815 17:57:08.098987 154793 storage/replica_command.go:280  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180815 17:57:08.104066 154585 storage/replica_command.go:280  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180815 17:57:08.111652 154849 storage/replica_command.go:280  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180815 17:57:08.118498 154880 storage/replica_command.go:280  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180815 17:57:08.126155 154863 storage/replica_command.go:280  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180815 17:57:08.132972 154920 storage/replica_command.go:280  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180815 17:57:08.138681 154933 storage/replica_command.go:280  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
W180815 17:57:08.175400 154357 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 17:57:08.181349 154357 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 17:57:08.181492 154357 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:08.181511 154357 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:08.183749 154357 server/config.go:496  [n?] 1 storage engine initialized
I180815 17:57:08.183771 154357 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 17:57:08.183780 154357 server/config.go:499  [n?] store 0: in-memory, size 0 B
W180815 17:57:08.183850 154357 gossip/gossip.go:1351  [n?] no incoming or outgoing connections
I180815 17:57:08.183880 154357 server/server.go:1400  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I180815 17:57:08.198211 155011 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:40607
I180815 17:57:08.198553 154736 gossip/server.go:219  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:35829}
I180815 17:57:08.198943 154357 server/node.go:697  [n?] connecting to gossip network to verify cluster ID...
I180815 17:57:08.198980 154357 server/node.go:722  [n?] node connected via gossip and verified as part of cluster "2b1f25af-ff88-4657-b6e8-c5105666374d"
I180815 17:57:08.199256 155036 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 17:57:08.200377 155035 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 17:57:08.200905 154357 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 17:57:08.201521 154357 server/node.go:428  [n?] new node allocated ID 2
I180815 17:57:08.201591 154357 gossip/gossip.go:383  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:35829" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1600-gaa1d0d2" started_at:1534355828201535307 
I180815 17:57:08.201633 154357 storage/stores.go:222  [n2] read 0 node addresses from persistent storage
I180815 17:57:08.201679 154357 storage/stores.go:241  [n2] wrote 1 node addresses to persistent storage
I180815 17:57:08.202407 155045 storage/stores.go:241  [n1] wrote 1 node addresses to persistent storage
I180815 17:57:08.203176 154357 server/node.go:672  [n2] bootstrapped store [n2,s2]
I180815 17:57:08.203282 154357 server/node.go:546  [n2] node=2: started with [] engine(s) and attributes []
I180815 17:57:08.203529 154357 server/status/recorder.go:652  [n2] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:08.203563 154357 server/server.go:1833  [n2] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 17:57:08.203625 154357 server/server.go:1535  [n2] starting https server at 127.0.0.1:38969 (use: 127.0.0.1:38969)
I180815 17:57:08.203642 154357 server/server.go:1537  [n2] starting grpc/postgres server at 127.0.0.1:35829
I180815 17:57:08.203656 154357 server/server.go:1538  [n2] advertising CockroachDB node at 127.0.0.1:35829
I180815 17:57:08.206096 155129 server/status/recorder.go:652  [n2,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:08.236326 154357 server/server.go:1620  [n2] done ensuring all necessary migrations have run
I180815 17:57:08.236357 154357 server/server.go:1623  [n2] serving sql connections
I180815 17:57:08.262879 154904 server/server_update.go:64  [n2] failed attempt to upgrade cluster version, error: node 2 not running (UNKNOWN), cannot determine version
I180815 17:57:08.271695 154906 sql/event_log.go:126  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:35829} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1600-gaa1d0d2 StartedAt:1534355828201535307 LocalityAddress:[]} ClusterID:2b1f25af-ff88-4657-b6e8-c5105666374d StartedAt:1534355828201535307 LastUp:1534355828201535307}
I180815 17:57:08.276808 155186 sql/event_log.go:126  [n1,client=127.0.0.1:48666,user=root] Event: "create_database", target: 52, info: {DatabaseName:test Statement:CREATE DATABASE IF NOT EXISTS test User:root}
I180815 17:57:08.284088 155023 rpc/nodedialer/nodedialer.go:92  [ct-client] connection to n2 established
I180815 17:57:08.284101 155228 storage/replica_command.go:280  [split,n1,s1,r22/1:/{Table/51-Max}] initiating a split of this range at key /Table/52 [r23]
I180815 17:57:08.289100 155186 sql/event_log.go:126  [n1,client=127.0.0.1:48666,user=root] Event: "create_table", target: 53, info: {TableName:test.public.t Statement:CREATE TABLE test.public.t (num INT) User:root}
I180815 17:57:08.291146 155235 storage/replica_command.go:280  [split,n1,s1,r23/1:/{Table/52-Max}] initiating a split of this range at key /Table/53 [r24]
I180815 17:57:08.307516 155269 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] auto-upgrade
I180815 17:57:08.313683 155268 util/stop/stopper.go:537  quiescing; tasks left:
1      storage.intentResolver: processing intents
1      node.Node: batch
1      kv.DistSender: sending partial batch
W180815 17:57:08.314027 154734 gossip/infostore.go:303  [n2] node unavailable; try another peer
I180815 17:57:08.314038 155268 util/stop/stopper.go:537  quiescing; tasks left:
1      storage.intentResolver: processing intents
1      node.Node: batch
--- FAIL: test/TestShowSessions (2.920s)
show_test.go:684: unexpected number of running sessions: 2, expected 1.
	Active sessions (results might have changed since the test checked):
	app: "", query: "SELECT active_queries, last_active_query, application_name FROM [SHOW CLUSTER SESSIONS]", last query: SELECT node_id, (now() - session_start)::FLOAT FROM [SHOW CLUSTER SESSIONS] WHERE application_name NOT LIKE 'internal-%'
------- Stdout: -------
W180815 17:57:07.840555 154357 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 17:57:07.854188 154357 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 17:57:07.854325 154357 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:07.854339 154357 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:07.856789 154357 server/config.go:496  [n?] 1 storage engine initialized
I180815 17:57:07.856823 154357 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 17:57:07.856833 154357 server/config.go:499  [n?] store 0: in-memory, size 0 B
I180815 17:57:07.869104 154357 server/node.go:373  [n?] **** cluster 2b1f25af-ff88-4657-b6e8-c5105666374d has been created
I180815 17:57:07.869185 154357 server/server.go:1398  [n?] **** add additional nodes by specifying --join=127.0.0.1:40607
I180815 17:57:07.869412 154357 gossip/gossip.go:383  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:40607" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1600-gaa1d0d2" started_at:1534355827869343827 
I180815 17:57:07.870450 154357 storage/store.go:1506  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180815 17:57:07.872955 154357 server/node.go:476  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=1, queries=0.00, writes=0.00, bytesPerReplica={p10=7103.00 p25=7103.00 p50=7103.00 p75=7103.00 p90=7103.00 pMax=7103.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180815 17:57:07.873118 154357 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180815 17:57:07.873222 154357 server/node.go:697  [n1] connecting to gossip network to verify cluster ID...
I180815 17:57:07.873258 154357 server/node.go:722  [n1] node connected via gossip and verified as part of cluster "2b1f25af-ff88-4657-b6e8-c5105666374d"
I180815 17:57:07.873293 154357 server/node.go:546  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180815 17:57:07.873769 154357 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:07.873829 154357 server/server.go:1833  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 17:57:07.878003 154561 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:07.890063 154357 server/server.go:1535  [n1] starting https server at 127.0.0.1:33349 (use: 127.0.0.1:33349)
I180815 17:57:07.890233 154357 server/server.go:1537  [n1] starting grpc/postgres server at 127.0.0.1:40607
I180815 17:57:07.890267 154357 server/server.go:1538  [n1] advertising CockroachDB node at 127.0.0.1:40607
I180815 17:57:07.918927 154596 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180815 17:57:07.925720 154569 storage/replica_command.go:280  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180815 17:57:07.934915 154472 storage/replica_command.go:280  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180815 17:57:07.945613 154612 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180815 17:57:07.952062 154247 storage/replica_command.go:280  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180815 17:57:07.960984 154627 storage/replica_command.go:280  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180815 17:57:07.977602 154636 storage/replica_command.go:280  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180815 17:57:07.978699 154476 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
W180815 17:57:07.984580 154624 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "split" id=ca8ea57d key=/Local/Range/System/tsd/RangeDescriptor rw=true pri=0.04340252 iso=SERIALIZABLE stat=PENDING epo=0 ts=1534355827.977704188,0 orig=1534355827.977704188,0 max=1534355827.977704188,0 wto=false rop=false seq=1
I180815 17:57:07.988519 154659 storage/replica_command.go:280  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180815 17:57:08.003157 154604 storage/replica_command.go:280  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180815 17:57:08.009692 154252 storage/replica_command.go:280  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180815 17:57:08.016497 154649 storage/replica_command.go:280  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180815 17:57:08.042777 154256 storage/replica_command.go:280  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180815 17:57:08.046632 154687 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
I180815 17:57:08.071168 154739 storage/replica_command.go:280  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180815 17:57:08.073688 154656 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180815 17:57:08.078018 154746 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180815 17:57:08.078638 154753 storage/replica_command.go:280  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180815 17:57:08.079904 154357 server/server.go:1620  [n1] done ensuring all necessary migrations have run
I180815 17:57:08.079943 154357 server/server.go:1623  [n1] serving sql connections
I180815 17:57:08.086056 154700 storage/replica_command.go:280  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180815 17:57:08.086571 154763 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180815 17:57:08.092285 154835 storage/replica_command.go:280  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180815 17:57:08.098779 154765 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:40607} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1600-gaa1d0d2 StartedAt:1534355827869343827 LocalityAddress:[]} ClusterID:2b1f25af-ff88-4657-b6e8-c5105666374d StartedAt:1534355827869343827 LastUp:1534355827869343827}
I180815 17:57:08.098987 154793 storage/replica_command.go:280  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180815 17:57:08.104066 154585 storage/replica_command.go:280  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180815 17:57:08.111652 154849 storage/replica_command.go:280  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180815 17:57:08.118498 154880 storage/replica_command.go:280  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180815 17:57:08.126155 154863 storage/replica_command.go:280  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180815 17:57:08.132972 154920 storage/replica_command.go:280  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180815 17:57:08.138681 154933 storage/replica_command.go:280  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
W180815 17:57:08.175400 154357 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 17:57:08.181349 154357 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 17:57:08.181492 154357 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:08.181511 154357 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 17:57:08.183749 154357 server/config.go:496  [n?] 1 storage engine initialized
I180815 17:57:08.183771 154357 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 17:57:08.183780 154357 server/config.go:499  [n?] store 0: in-memory, size 0 B
W180815 17:57:08.183850 154357 gossip/gossip.go:1351  [n?] no incoming or outgoing connections
I180815 17:57:08.183880 154357 server/server.go:1400  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I180815 17:57:08.198211 155011 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:40607
I180815 17:57:08.198553 154736 gossip/server.go:219  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:35829}
I180815 17:57:08.198943 154357 server/node.go:697  [n?] connecting to gossip network to verify cluster ID...
I180815 17:57:08.198980 154357 server/node.go:722  [n?] node connected via gossip and verified as part of cluster "2b1f25af-ff88-4657-b6e8-c5105666374d"
I180815 17:57:08.199256 155036 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 17:57:08.200377 155035 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 17:57:08.200905 154357 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 17:57:08.201521 154357 server/node.go:428  [n?] new node allocated ID 2
I180815 17:57:08.201591 154357 gossip/gossip.go:383  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:35829" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1600-gaa1d0d2" started_at:1534355828201535307 
I180815 17:57:08.201633 154357 storage/stores.go:222  [n2] read 0 node addresses from persistent storage
I180815 17:57:08.201679 154357 storage/stores.go:241  [n2] wrote 1 node addresses to persistent storage
I180815 17:57:08.202407 155045 storage/stores.go:241  [n1] wrote 1 node addresses to persistent storage
I180815 17:57:08.203176 154357 server/node.go:672  [n2] bootstrapped store [n2,s2]
I180815 17:57:08.203282 154357 server/node.go:546  [n2] node=2: started with [] engine(s) and attributes []
I180815 17:57:08.203529 154357 server/status/recorder.go:652  [n2] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:08.203563 154357 server/server.go:1833  [n2] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 17:57:08.203625 154357 server/server.go:1535  [n2] starting https server at 127.0.0.1:38969 (use: 127.0.0.1:38969)
I180815 17:57:08.203642 154357 server/server.go:1537  [n2] starting grpc/postgres server at 127.0.0.1:35829
I180815 17:57:08.203656 154357 server/server.go:1538  [n2] advertising CockroachDB node at 127.0.0.1:35829
I180815 17:57:08.206096 155129 server/status/recorder.go:652  [n2,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 17:57:08.236326 154357 server/server.go:1620  [n2] done ensuring all necessary migrations have run
I180815 17:57:08.236357 154357 server/server.go:1623  [n2] serving sql connections
I180815 17:57:08.262879 154904 server/server_update.go:64  [n2] failed attempt to upgrade cluster version, error: node 2 not running (UNKNOWN), cannot determine version
I180815 17:57:08.271695 154906 sql/event_log.go:126  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:35829} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1600-gaa1d0d2 StartedAt:1534355828201535307 LocalityAddress:[]} ClusterID:2b1f25af-ff88-4657-b6e8-c5105666374d StartedAt:1534355828201535307 LastUp:1534355828201535307}
I180815 17:57:08.276808 155186 sql/event_log.go:126  [n1,client=127.0.0.1:48666,user=root] Event: "create_database", target: 52, info: {DatabaseName:test Statement:CREATE DATABASE IF NOT EXISTS test User:root}
I180815 17:57:08.284088 155023 rpc/nodedialer/nodedialer.go:92  [ct-client] connection to n2 established
I180815 17:57:08.284101 155228 storage/replica_command.go:280  [split,n1,s1,r22/1:/{Table/51-Max}] initiating a split of this range at key /Table/52 [r23]
I180815 17:57:08.289100 155186 sql/event_log.go:126  [n1,client=127.0.0.1:48666,user=root] Event: "create_table", target: 53, info: {TableName:test.public.t Statement:CREATE TABLE test.public.t (num INT) User:root}
I180815 17:57:08.291146 155235 storage/replica_command.go:280  [split,n1,s1,r23/1:/{Table/52-Max}] initiating a split of this range at key /Table/53 [r24]
I180815 17:57:08.307516 155269 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] auto-upgrade
I180815 17:57:08.313683 155268 util/stop/stopper.go:537  quiescing; tasks left:
1      storage.intentResolver: processing intents
1      node.Node: batch
1      kv.DistSender: sending partial batch
W180815 17:57:08.314027 154734 gossip/infostore.go:303  [n2] node unavailable; try another peer
I180815 17:57:08.314038 155268 util/stop/stopper.go:537  quiescing; tasks left:
1      storage.intentResolver: processing intents
1      node.Node: batch
------- Stdout: -------
W180815 18:19:02.736458 157230 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 18:19:02.770384 157230 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 18:19:02.770932 157230 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 18:19:02.771008 157230 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 18:19:02.775577 157230 server/config.go:496  [n?] 1 storage engine initialized
I180815 18:19:02.775824 157230 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 18:19:02.775947 157230 server/config.go:499  [n?] store 0: in-memory, size 0 B
I180815 18:19:02.831175 157230 server/node.go:373  [n?] **** cluster f4cd7a29-b689-4d55-a415-14bf983ce6de has been created
I180815 18:19:02.831445 157230 server/server.go:1398  [n?] **** add additional nodes by specifying --join=127.0.0.1:42937
I180815 18:19:02.832756 157230 gossip/gossip.go:383  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:42937" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1600-gaa1d0d2" started_at:1534357142832342734 
I180815 18:19:02.854313 157230 storage/store.go:1506  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180815 18:19:02.855632 157230 server/node.go:476  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7103.00 p25=7103.00 p50=7103.00 p75=7103.00 p90=7103.00 pMax=7103.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180815 18:19:02.856297 157230 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180815 18:19:02.857626 157230 server/node.go:697  [n1] connecting to gossip network to verify cluster ID...
I180815 18:19:02.857971 157230 server/node.go:722  [n1] node connected via gossip and verified as part of cluster "f4cd7a29-b689-4d55-a415-14bf983ce6de"
I180815 18:19:02.858337 157230 server/node.go:546  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180815 18:19:02.859901 157230 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 18:19:02.860104 157230 server/server.go:1833  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 18:19:02.861761 157554 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 18:19:02.862373 157230 server/server.go:1535  [n1] starting https server at 127.0.0.1:36855 (use: 127.0.0.1:36855)
I180815 18:19:02.862620 157230 server/server.go:1537  [n1] starting grpc/postgres server at 127.0.0.1:42937
I180815 18:19:02.862700 157230 server/server.go:1538  [n1] advertising CockroachDB node at 127.0.0.1:42937
I180815 18:19:02.888044 157356 storage/replica_command.go:280  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180815 18:19:03.034060 157576 storage/replica_command.go:280  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180815 18:19:03.107479 157296 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180815 18:19:03.127753 157329 storage/replica_command.go:280  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180815 18:19:03.217829 157361 storage/replica_command.go:280  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180815 18:19:03.315103 157666 storage/replica_command.go:280  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
W180815 18:19:03.324586 157640 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=b343f882 key=/Table/SystemConfigSpan/Start rw=true pri=0.00330894 iso=SERIALIZABLE stat=PENDING epo=0 ts=1534357143.180233943,0 orig=1534357143.180233943,0 max=1534357143.180233943,0 wto=false rop=false seq=12
I180815 18:19:03.385703 157606 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180815 18:19:03.414519 157593 storage/replica_command.go:280  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180815 18:19:03.514454 157624 storage/replica_command.go:280  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180815 18:19:03.551180 157669 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180815 18:19:03.604760 157654 storage/replica_command.go:280  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180815 18:19:03.663829 157629 storage/replica_command.go:280  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180815 18:19:03.722227 157598 storage/replica_command.go:280  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180815 18:19:03.784090 157597 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
I180815 18:19:03.785030 157747 storage/replica_command.go:280  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180815 18:19:03.853723 157707 storage/replica_command.go:280  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180815 18:19:03.924186 157717 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180815 18:19:03.935061 157752 storage/replica_command.go:280  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180815 18:19:03.967290 157719 rpc/nodedialer/nodedialer.go:92  [consistencyChecker,n1,s1,r1/1:/{Min-System/}] connection to n1 established
I180815 18:19:04.000286 157711 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180815 18:19:04.007394 157799 storage/replica_command.go:280  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180815 18:19:04.044485 157230 server/server.go:1620  [n1] done ensuring all necessary migrations have run
I180815 18:19:04.044786 157230 server/server.go:1623  [n1] serving sql connections
I180815 18:19:04.079337 157744 storage/replica_command.go:280  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180815 18:19:04.100566 157828 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:42937} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1600-gaa1d0d2 StartedAt:1534357142832342734 LocalityAddress:[]} ClusterID:f4cd7a29-b689-4d55-a415-14bf983ce6de StartedAt:1534357142832342734 LastUp:1534357142832342734}
I180815 18:19:04.101863 157826 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180815 18:19:04.153459 157844 storage/replica_command.go:280  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180815 18:19:04.221018 157836 storage/replica_command.go:280  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180815 18:19:04.284151 157808 storage/replica_command.go:280  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180815 18:19:04.336205 157882 storage/replica_command.go:280  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180815 18:19:04.396861 157887 storage/replica_command.go:280  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180815 18:19:04.451274 157900 storage/replica_command.go:280  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
W180815 18:19:04.575093 157230 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180815 18:19:04.614094 157230 server/server.go:827  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180815 18:19:04.614621 157230 base/addr_validation.go:260  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 18:19:04.614702 157230 base/addr_validation.go:300  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180815 18:19:04.619147 157230 server/config.go:496  [n?] 1 storage engine initialized
I180815 18:19:04.619343 157230 server/config.go:499  [n?] RocksDB cache size: 128 MiB
I180815 18:19:04.619371 157230 server/config.go:499  [n?] store 0: in-memory, size 0 B
W180815 18:19:04.619726 157230 gossip/gossip.go:1351  [n?] no incoming or outgoing connections
I180815 18:19:04.620312 157230 server/server.go:1400  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I180815 18:19:04.691939 157951 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:42937
I180815 18:19:04.693438 158034 gossip/server.go:219  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:43901}
I180815 18:19:04.698668 157230 server/node.go:697  [n?] connecting to gossip network to verify cluster ID...
I180815 18:19:04.699525 157230 server/node.go:722  [n?] node connected via gossip and verified as part of cluster "f4cd7a29-b689-4d55-a415-14bf983ce6de"
I180815 18:19:04.703500 158037 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 18:19:04.710580 158036 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 18:19:04.717899 157230 kv/dist_sender.go:345  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180815 18:19:04.728633 157230 server/node.go:428  [n?] new node allocated ID 2
I180815 18:19:04.729313 157230 gossip/gossip.go:383  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:43901" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > build_tag:"v2.1.0-alpha.20180702-1600-gaa1d0d2" started_at:1534357144728966999 
I180815 18:19:04.730239 157230 storage/stores.go:222  [n2] read 0 node addresses from persistent storage
I180815 18:19:04.730700 157230 storage/stores.go:241  [n2] wrote 1 node addresses to persistent storage
I180815 18:19:04.734948 158011 storage/stores.go:241  [n1] wrote 1 node addresses to persistent storage
I180815 18:19:04.758757 157230 server/node.go:672  [n2] bootstrapped store [n2,s2]
I180815 18:19:04.761616 157230 server/node.go:546  [n2] node=2: started with [] engine(s) and attributes []
I180815 18:19:04.763052 157230 server/status/recorder.go:652  [n2] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 18:19:04.763218 157230 server/server.go:1833  [n2] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180815 18:19:04.765049 158148 server/status/recorder.go:652  [n2,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180815 18:19:04.766349 157230 server/server.go:1535  [n2] starting https server at 127.0.0.1:36425 (use: 127.0.0.1:36425)
I180815 18:19:04.766581 157230 server/server.go:1537  [n2] starting grpc/postgres server at 127.0.0.1:43901
I180815 18:19:04.766636 157230 server/server.go:1538  [n2] advertising CockroachDB node at 127.0.0.1:43901
I180815 18:19:04.773730 157230 server/server.go:1620  [n2] done ensuring all necessary migrations have run
I180815 18:19:04.774030 157230 server/server.go:1623  [n2] serving sql connections
I180815 18:19:04.880278 158166 server/server_update.go:67  [n2] no need to upgrade, cluster already at the newest version
I180815 18:19:04.883766 158168 sql/event_log.go:126  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:43901} Attrs: Locality: ServerVersion:2.0-11 BuildTag:v2.1.0-alpha.20180702-1600-gaa1d0d2 StartedAt:1534357144728966999 LocalityAddress:[]} ClusterID:f4cd7a29-b689-4d55-a415-14bf983ce6de StartedAt:1534357144728966999 LastUp:1534357144728966999}
I180815 18:19:04.941285 158184 sql/event_log.go:126  [n1,client=127.0.0.1:41292,user=root] Event: "create_database", target: 52, info: {DatabaseName:test Statement:CREATE DATABASE IF NOT EXISTS test User:root}
I180815 18:19:04.941417 158048 rpc/nodedialer/nodedialer.go:92  [ct-client] connection to n2 established
I180815 18:19:04.945522 158196 storage/replica_command.go:280  [split,n1,s1,r22/1:/{Table/51-Max}] initiating a split of this range at key /Table/52 [r23]
I180815 18:19:05.011544 158184 sql/event_log.go:126  [n1,client=127.0.0.1:41292,user=root] Event: "create_table", target: 53, info: {TableName:test.public.t Statement:CREATE TABLE test.public.t (num INT) User:root}
I180815 18:19:05.031140 158216 storage/replica_command.go:280  [split,n1,s1,r23/1:/{Table/52-Max}] initiating a split of this range at key /Table/53 [r24]
W180815 18:19:05.077307 158184 sql/crdb_internal.go:900  [n1,client=127.0.0.1:41292,user=root] failed to dial into node 2 (LIVE): initial connection heartbeat failed: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:43901: connect: connection refused"
I180815 18:19:05.081695 158306 util/stop/stopper.go:537  quiescing; tasks left:
1      node.Node: batch
1      [async] transport racer
1      [async] storage.split: processing replica
1      [async] kv.TxnCoordSender: heartbeat loop
I180815 18:19:05.081869 157566 kv/transport_race.go:67  transport race promotion: ran 46 iterations on up to 947 requests
I180815 18:19:05.082643 158306 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] storage.split: processing replica
1      [async] kv.TxnCoordSender: heartbeat loop
I180815 18:19:05.083392 158306 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] storage.split: processing replica
W180815 18:19:05.083542 158216 internal/client/txn.go:514  [split,n1,s1,r23/1:/{Table/52-Max}] failure aborting transaction: node unavailable; try another peer; abort caused by: node unavailable; try another peer
E180815 18:19:05.084468 158216 storage/queue.go:788  [split,n1,s1,r23/1:/{Table/52-Max}] unable to split [n1,s1,r23/1:/{Table/52-Max}] at key "/Table/53": split at key /Table/53 failed: node unavailable; try another peer

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/show_test.go
+++ b/pkg/sql/show_test.go
@@ -619,10 +619,10 @@ func TestShowSessions(t *testing.T) {
 	sqlutils.CreateTable(t, conn, "t", "num INT", 0, nil)
 
 	// We'll skip "internal" sessions, as those are unpredictable.
-	const showSessions = `
+	var showSessions = fmt.Sprintf(`
 	select node_id, (now() - session_start)::float from
-		[show cluster sessions] where application_name not like 'internal-%'
-	`
+		[show cluster sessions] where application_name not like '%s%%'
+	`, sql.InternalAppNamePrefix)
 
 	rows, err := conn.Query(showSessions)
 	if err != nil {

